android/source/VideoLocker/src/org/edx/mobile/module/validate/ValidationUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.edx.mobile.module.validate;
+
+import com.facebook.internal.*;
+
+import java.util.Collection;
+
+/**
+ * Created by rohan on 1/23/15.
+ */
+public class ValidationUtil {
+
+    /**
+     * Returns true if given object is null, false otherwise.
+     * @param obj
+     * @return
+     */
+    public static boolean isNull(Object obj) {
+        return (obj == null);
```

Comment:
This doesn't seem any more useful than doing a null check directly and is less idiomatic, since people will have to look at this function to verify it does what they think it does?